orc8r/protos/mconfig/mconfigs.proto

```diff
@@ -45,6 +45,8 @@ message SharedSentryConfig {
   // sample_rate should be a number between 0 (0% of errors sent) and 1 (100%
   // of errors sent)
   float sample_rate = 4;
+  // message patterns that are excluded from Sentry (regex substring match)
+  repeated string sentry_excluded_errors = 5;
```

Comment:
We don't need the `sentry_` prefix as it's already under a message called sentry config. I think we can call it something like `exclusion_patterns`? :D

Reply:
Changed it in both interfaces.